apps/users/schemas.py

```diff
@@ -139,39 +139,53 @@ class DeleteUserIn(BaseInSchema):
     user_id: int
 
 
-class StudentCheckExistanceIn(BaseInSchema):   # TODO spelling error
-    full_name: str
-    telephone_number: str
+class StudentCheckExistenceIn(BaseInSchema):
+    last_name: str = Field(default="Petrenko", max_length=50)
+    first_name: str = Field(default="Petro", max_length=50)
+    telephone_number: str = Field(default="380979889988", max_length=12, min_length=12)
+
+    @validator('last_name')
+    def validate_last_name(cls, value):
+        if not value:
+            raise ValueError("The student's surname is mandatory!")
+        if not value.istitle():
+            raise ValueError("The last name first letter must be uppercase!")
+        return value
+
+    @validator('first_name')
+    def validate_first_name(cls, value):
+        if not value:
+            raise ValueError("The student's name is mandatory!")
+        if not value.istitle():
+            raise ValueError("The name first letter must be uppercase!")
+        return value
+
+    @validator('telephone_number')
+    def validate_telephone_number(cls, value):
+        if not value.isdigit():
+            raise ValueError('The phone number must consist of digits!')
+        return value
 
 
-class StudentCheckExistanceOut(BaseOutSchema):
+class StudentCheckExistenceOut(BaseOutSchema):
     student: int
     token: str
     expires: datetime
 
 
-class CreateStudentIn(BaseInSchema):
-    full_name: str
-    telephone_number: str
+class CreateStudentIn(StudentCheckExistenceIn):
```

Comment:
after inheriting from class StudentCheckExistenceIn validation of field first, last names and telephone doesn't work